core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -103,6 +103,12 @@ oppia.factory('CollectionNodeObjectFactory', [
         explorationSummaryBackendObject);
     };
 
+    CollectionNode.prototype.getCapitalizedObjective = function(
+        explorationObjective) {
```

Comment:
Okay. Should I use angular.copy or can I just use this._explorationSummaryObject.objective